config/initializers/mime_types.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
```

Comment:
?

Reply:
We don't plan to use mime classes so deleted